docs/3.0/actions.md

```diff
@@ -103,6 +103,40 @@ def handle(query:, fields:, current_user:, resource:, **args)
 end
 ```
 
+## Passing Params to the Action Show Page
+When working with resource index or show views, it's often necessary to pass parameters to an action.
+
+Example
```

Comment:
I don't think we need the "Example" copy if we have "consider the following example" (which is great copy! GJ!)

Reply:
Ok, I will delete it. Thanks for the review.